CNV-45186: Issue in file virt-configuring-storage-profile.adoc (#87971)
openshift/openshift-docs: modules/virt-viewing-automatically-created-storage-profiles.adoc

// Module included in the following assemblies:
//
// * virt/storage/virt-configuring-storage-profile.adoc

:_mod-docs-content-type: PROCEDURE
[id="virt-viewing-automatically-created-storage-profiles_{context}"]
= Viewing automatically created storage profiles

> 

Reviewer: Missing .Procedure here

The system creates storage profiles for each storage class automatically.

> . To view the list of storage profiles, run the following command:

Reviewer 2: Suggestion. Up to you.
```suggestion
. View the list of storage profiles by running the following command:
```

+
[source,terminal]
----

> oc get storageprofile

Reviewer: ```suggestion
$ oc get storageprofile
```

----

> . To print the details of a particular storage profile, run the following command:

Reviewer 2: ```suggestion
. Fetch the details of a particular storage profile by running the following command:
```

+
[source,terminal]
----

> oc describe storageprofile <name>

Reviewer: ```suggestion
$ oc describe storageprofile <name>
```

----

> .Example storage profile details

Reviewer: Is this an example output? if so, it should be lined up with the second command with a "+" sign. Eg: 

```suggestion
+
.Example storage profile details
```

[source,yaml]
----

> $ oc describe  storageprofile ocs-storagecluster-ceph-rbd-virtualization

Reviewer: This seems repetitive with the above command and shouldn't be included in the YAML output.

              cdi.kubevirt.io=
Annotations:  <none>
API Version:  cdi.kubevirt.io/v1beta1
Kind:         StorageProfile
Metadata:
  Creation Timestamp:  2023-11-13T07:58:02Z
  Generation:          2
  Owner References:
    API Version:           cdi.kubevirt.io/v1beta1
    Block Owner Deletion:  true
    Controller:            true
    Kind:                  CDI
    Name:                  cdi-kubevirt-hyperconverged
    UID:                   2d6f169a-382c-4caf-b614-a640f2ef8abb
  Resource Version:        4186799537
  UID:                     14aef804-6688-4f2e-986b-0297fd3aaa68
Spec:
Status:
  Claim Property Sets: <1>
    accessModes:
      ReadWriteMany
    volumeMode:  Block
    accessModes:
      ReadWriteOnce
    volumeMode:  Block
    accessModes:
      ReadWriteOnce
    volumeMode:                   Filesystem
  Clone Strategy:                  csi-clone <2>
  Data Import Cron Source Format:  snapshot <3>
  Provisioner:                     openshift-storage.rbd.csi.ceph.com
  Snapshot Class:                  ocs-storagecluster-rbdplugin-snapclass
  Storage Class:                   ocs-storagecluster-ceph-rbd-virtualization
Events:                            <none>
----

> <1> `Claim Property Sets` is an ordered list of `AccessMode`/`VolumeMode` pairs, which describe the PVC modes that will be used to provision VM disks.

Reviewer: In general, stay away from "future tense": https://spaces.redhat.com/display/CCSENABLE/Future+tense

```suggestion
<1> `Claim Property Sets` is an ordered list of `AccessMode`/`VolumeMode` pairs, which describe the PVC modes that are used to provision VM disks.
```
